aether-ui/aether/ui/assets/apps/pipeline/components/PipelineCard.jsx

```diff
@@ -29,14 +29,19 @@ import ContractCard from './ContractCard'
 import { selectPipeline } from '../redux'
 
 class PipelineCard extends Component {
+  onPipelineSelect (pipeline) {
+    this.props.selectPipeline(pipeline.id)
+    this.props.history.push(`/${pipeline.id}`)
+  }
```

Comment:
This is triggered by `componentDidUpdate` method in `PipelineList` component. You don't need to add this here.

Reply:
Ref. above comment